### PR TITLE
feat: remove monitoring

### PR DESCRIPTION
**Pull request by kaldorn**
Please read [CONTRIBUTING.md](CONTRIBUTING.md) for additional information on contributing to this repository!

## What this PR does / why we need it
This removes the monitoring templates from stencil-golang

## Jira ID

[XX-XX]

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -2,22 +2,26 @@ name: github.com/getoutreach/stencil-golang
...
+      type: number
+    description: Overrides the low traffic threshold in the "http_latency_high" terraform module in monitoring/datadog.tf.
+  terraform.datadog.http.percentiles.lowTraffic:
+    schema:
+      type: number
+    description: Overrides the low traffic percentile in the "http_success_rate_low" terraform module in monitoring/datadog.tf.
+  terraform.datadog.podRestart.thresholds.lowCount:
+    schema:
+      type: number
+    description: Overrides the pod restarts threshold in the "pod_restarts" terraform module in monitoring/datadog.tf.
```
Comment (malept) [inline manifest.yaml:24]: question: are these still here because they're referenced in other Stencil modules?

Comment (kaldorn) [reply manifest.yaml:24]: they're used in the canary stuff in jsonnet

Comment (kaldorn) [reply manifest.yaml:24]: will add a comment

Comment (malept) [reply manifest.yaml:25]: At some point we should move these values to a more relevant fully qualified configuration option name.

**Review by malept — APPROVED** [review]

File: manifest.yaml (hunk 1 above)
Comment (kaldorn) [reply manifest.yaml:25]: I strongly agree.

**Pull request merged**